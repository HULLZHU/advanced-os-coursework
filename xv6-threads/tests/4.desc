testing whether PIDs returned by clone() and join() system calls are correct