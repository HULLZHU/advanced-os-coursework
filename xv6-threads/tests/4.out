XV6_TEST_OUTPUT : Correct PIDs returned by clone() and join()
